core/src/main/resources/error/error-classes.json

```diff
@@ -63,5 +63,67 @@
   "WRITING_JOB_ABORTED" : {
     "message" : [ "Writing job aborted" ],
     "sqlState" : "40000"
+  },
+  "UNSUPPORTED_CHANGE_COLUMN" : {
+    "message" : [ "Please add an implementation for a column change here" ],
+    "sqlState" : "46110"
+  },
+  "LOGICAL_HINT_OPERATOR_NOT_REMOVE_DURING_ANALYSIS" : {
+    "message" : [ "Internal error: logical hint operator should have been removed during analysis" ],
+    "sqlState" : "42000"
+  },
+  "CAN_NOT_EVALUATE_EXPRESSION" : {
```

Comment:
Nit: I think `cannot` is more common than `can not`.

Reply:
done